tidy3d/web/environment.py

```diff
@@ -76,6 +76,8 @@ def __init__(self):
             self._current = dev
         elif env_key == "uat":
             self._current = uat
+        else:
+            self._current = prod
```

Comment:
The issue which this PR is trying to solver is that current tidy3d will throw ` 'Environment' object has no attribute '_current'` when user set `TIDY3D_ENV=prod`.

Reply:
Yea that makes sense, I was just thinking that in the logic, it would be good to distinguish between
a. User selecting `env = "prod"
b. User not selecting an `env` and getting prod as a result.
c. User inputs an unsupported env key (warn and get prod).

Otherwise a user could supply something eg `env = "development"` and silently get `prod` without realizing. I think the latest change fixes this.